scripts/imgtool/keys/ecdsa.py

```diff
@@ -102,37 +102,57 @@ def generate():
     def _get_public(self):
         return self.key.public_key()
 
-    def _build_minimal_ecdsa_privkey(self, der):
+    def _build_minimal_ecdsa_privkey(self, der, format):
+        if (format == serialization.PrivateFormat.OpenSSH):
+            print(os.path.basename(__file__) + \
+                ": Warning: --minimal is supported only for PKCS8 or TraditionalOpenSSL formats")
+            return bytearray(der)
         '''
         Builds a new DER that only includes the EC private key, removing the
         public key that is added as an "optional" BITSTRING.
         '''
-        offset_PUB = 68
         EXCEPTION_TEXT = "Error parsing ecdsa key. Please submit an issue!"
-        if der[offset_PUB] != 0xa1:
-            raise ECDSAUsageError(EXCEPTION_TEXT)
-        len_PUB = der[offset_PUB + 1]
-        b = bytearray(der[:-offset_PUB])
-        offset_SEQ = 29
-        if b[offset_SEQ] != 0x30:
-            raise ECDSAUsageError(EXCEPTION_TEXT)
-        b[offset_SEQ + 1] -= len_PUB
-        offset_OCT_STR = 27
-        if b[offset_OCT_STR] != 0x04:
-            raise ECDSAUsageError(EXCEPTION_TEXT)
-        b[offset_OCT_STR + 1] -= len_PUB
-        if b[0] != 0x30 or b[1] != 0x81:
-            raise ECDSAUsageError(EXCEPTION_TEXT)
-        b[2] -= len_PUB
+        if (format == serialization.PrivateFormat.PKCS8):
+            offset_PUB = 68 # where the context specific TLV starts (tag: 0xA1)
```

Comment:
Please add another space before the `#`. see: https://peps.python.org/pep-0008/#inline-comments

Reply:
Done as part of PEP8 validation.